src/functions/outputs/tableOutput.ts

```diff
@@ -115,10 +108,9 @@ export function tableOutput(
             return `Sorry we're out of supplies. Missing: ${missingIngredientsNames.join(', ')}`;
         } else if (!anyoneIsAlergic && anyoneOrderedUnavailableFood) {
             return informationAboutUnavailableFood.find(x => x.length > 0);
-        } else if (!anyoneIsAlergic && !anyoneOrderedUnavailableFood) {
+        } else if (!anyoneIsAlergic && !anyoneOrderedUnavailableFood && commandAndParameters.flag.toLowerCase() != 'Pooled'.toLowerCase()) {
             const whoCouldntPay: ICustomerAlergies[] = [];
```

Comment:
Instead of such dirty comparing (which can lead to errors) you should use enums.